Library/Homebrew/extend/os/linux/formula.rb

```diff
@@ -1,12 +1,10 @@
 # typed: true # rubocop:disable Sorbet/StrictSigil
 # frozen_string_literal: true
 
-class Formula
-  undef shared_library
-  undef loader_path
-  undef deuniversalize_machos
-  undef add_global_deps_to_spec
-  undef valid_platform?
+module FormulaLinux
```

Comment:
Maybe it's crazy or more confusing than it needs to be but: I think it might be nice to put all new classes/modules in this PR (and, perhaps with RuboCop, in all future PRs) under the `Homebrew` module whenever possible. Thoughts?